### PR TITLE
Update Dockerfile to latest standard

### PR DESCRIPTION
**Pull request by ddgenome**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,54 +4,50 @@ LABEL maintainer="Atomist <docker@atomist.com>"
...
 
 RUN git config --global user.email "bot@atomist.com" \
-    &&  git config --global user.name "Atomist Bot"
+    && git config --global user.name "Atomist Bot"
 
-RUN curl -sL https://deb.nodesource.com/setup_9.x | bash - \
-    && apt-get update \
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
     && apt-get install -y nodejs \
-    && npm install -g npm@6.4.1 \
```
Comment (cdupuis) [inline Dockerfile]: Why did you remove this? I kinda liked that we always got the latest npm version installed in our image.

Comment (ddgenome) [reply Dockerfile]: I removed it because it wasn't necessary as the version of NPM shipped with Node.js has everything we need and is likely better tested and more stable than the latest release of NPM, which over the last year has had several issues.

It simplifies things but I understand your thinking. I can add it back in if you wish.
